akka-docs/src/main/paradox/howto.md

```diff
@@ -163,4 +163,10 @@ Finally we are able to execute an actor and receive the results or exceptions.
 
 @@snip [SupervisedAskSpec.java]($code$/java/jdocs/pattern/SupervisedAskSpec.java)
 
-@@@
\ No newline at end of file
+@@@
+
+## Scalable Distributed Eventsourcing and CQRS
+
+The Lagom Framework encodes many best practices in combining @ref:[Akka Persistence](persistence.md) and @ref:[Akka Persistence Query](persistence-query.md) 
+with @ref:[Cluster Sharding](cluster-sharding.md) to build scalable and resilient systems with Event Sourcing and CQRS.
```

Comment:
I'd rather link specifically to the read side feature they have since it is an addition over what akka offers?

Reply:
It seems I forgot the link completely here. Looking through the docs we don't actually have a persistence+sharding+persistence-query part, just the separate reference docs, so I think linking to the entire persistence API side of lagom is fine.

I think I can rewrite this though into something more along the lines of "You can combine these yourself, but here is a prepackaged-solution encoding many best practices you'd have to take care of yourself: lagom persistence docs"

Reply:
👍

Reply:
Perhaps some more links similar to that from Akka Persistence and Persistence Query

Reply:
I think we stick to `Event Sourcing` (two words, both capitalised). (see section title)

About the discussion above: we could link to https://www.lagomframework.com/documentation/1.4.x/java/ES_CQRS.html (which is in the `core concepts` section) or to https://www.lagomframework.com/documentation/1.4.x/java/PersistentEntity.html (which is the main page on the ref guide and introduces `Aggregate Root` and other concepts, with sample code and in more detail).

Reply:
I'll try to find some more places for something like this. Thanks for the link ignasi, I'll use the core concepts one.